Beta release notes 0.112 (#13822)
home-assistant/home-assistant.github.io: source/_posts/2020-06-24-release-112.markdown

---
layout: post
title: "0.112: Beta release notes"
description: "0.112: Beta release notes"
date: 2020-06-24 00:00:00
date_formatted: "July 1, 2020"
author: Franck Nijhof
author_twitter: frenck
comments: true
categories: Release-Notes
og_image: /images/blog/2020-06-0.112/social.png
---

<a href='/integrations/#version/0.112'><img src='/images/blog/2020-06-0.112/social.png' style='border: 0;box-shadow: none;'></a>

> And another 3 week passed! Home Assistant Core 0.112.0!

Reviewer: ```suggestion
And another 3 weeks have passed! Home Assistant Core 0.112.0!
```

Please note, these are the beta release notes and might still change during
the beta week.

## Important upgrade notice!

Let's start with an important notice for upgrading to Home Assistant Core 0.112.

> This release has changes to the database format of Home Assistant. On upgrade,

Reviewer 2: This would also indicate that rolling back to an earlier version could be difficult / impossible. Should we warn for this?

it will be migrated to the new format.

> 

Reviewer 2: This would also indicate that rolling back to an earlier version could be difficult / impossible. Should we warn for this?

Author: That isn't an issue, hence it is not mentioned.

Reviewer 2: Great!

Depending on the size of your database and the performance of the hardware you
run Home Assistant on, this migration process could add additional time to the

> first time starting after the upgrade. For most of us, this add just a couple of

Reviewer: ```suggestion
first time starting after the upgrade. For most of us, this adds just a couple of
```

minutes, but if you have an enormous database (for example, 30+ GiB), it might
take an hour or maybe even two.

So, don't panic if Home Assistant doesn't come up immediately.

## Logbook and History 🚀

The reason for the above-mentioned database change can be found in the
Logbook & History. Thanks to [@bdraco], who has been improving this part
of Home Assistant the past weeks.

> As a result of his effort, this History & Logbook are now lightning fast. ⚡️

Reviewer: ```suggestion
As a result of his effort, the History & Logbook are now lightning fast. ⚡️
```

Not a tiny bit faster, but a whole new experience compared to how it used to
be. Fantastic work there [@bdraco]!

## Logbook and History now have a date/time range picker

So, [@bramkragten] stepped on the Logbook & History train and made the logbook
and history panels a lot more flexible.

You no longer have to choose between a fixed period to show, but can define

> the period you want to see data from yourself.

Reviewer: ```suggestion
the period you want to see data for yourself.
```

```yaml
condition:
  - platform: state
    entity_id: alarm_control_pane.home
    state:
      - armed_home
      - armed_away
```

And this condition is now also passing when Frenck is at home or at work
(fake, of course, since Frenck works from his home).

```yaml
condition:
  - platform: zone
    entity_id: device_tracker.frenck
    zone:
      - zone.home
      - zone.work
```

Or combine! The following condition would pass if both Frenck and Daphne
are either at home or in the work zone.

```yaml
condition:
  platform: zone
  entity_id:
    - device_tracker.frenck
    - device_tracker.daphne
  zone:
    - zone.home
    - zone.work
```

## Integration specific panels are now on the integrations page

Some integrations, like ZHA, Z-Wave and MQTT have their own panels or dev tools.
For things like pairing devices or publishing MQTT messages.

These used to be on the configuration page or development tools, which was weird
because the integration settings would be on the integrations page. ZHA also had
a lot of device settings in its own panel, resulting in a lot of duplicate
functionality spread across the Home Assistant interface.

> These panels and tools now moved to the integrations pages, directly with

Reviewer: ```suggestion
These panels and tools have now moved to the integrations pages, directly within
```

the integration that provides those. You can find them on the card of the
integration at the `Configure` button.

<p class='img'>
<img src='/images/blog/2020-06-0.112/configure-mqtt.png' alt='Screenshot of the MQTT configure button'></a>
Screenshot of the MQTT configure button.
</p>

The device-specific settings are now available on the device page, so we now

> have 1 place to go for information and setting for devices.

Reviewer: ```suggestion
have 1 place to go for information and settings for devices.
```

<p class='img'>
<img src='/images/blog/2020-06-0.112/zha-device.png' alt='Screenshot of a ZHA device'></a>
Screenshot of a ZHA device.
</p>

In the process, we also cleaned some things ups, like adding a device in ZHA.
If you change the name of the device while adding it, the entities and entity
IDs are named accordingly.

## New home for the logs and information pages

More moving this release, the logs and information pages used to be in the

> development tools panel. But they didn't really belong there. They aren't

Reviewer: ```suggestion
development tools panel, but they didn't really belong there. They aren't
```
